bentoml/server/gunicorn_config.py

```diff
@@ -54,3 +55,4 @@ def worker_abort(worker):
 def post_worker_init(worker):
     worker.log.debug('Unregistering usage tracking in worker process')
     atexit.unregister(track_server_stop)
+    atexit.unregister(_exit_function)  # Shutting down Gunicorn gracefully
```

Comment:
any reason for this? I thought  _exit_function is already in the atexit callback list by default?

Reply:
https://github.com/benoitc/gunicorn/issues/1391